From 74751935e613523e2db96a626c459d583d6bc11b Mon Sep 17 00:00:00 2001
From: Jinzhe Zeng <jinzhe.zeng@rutgers.edu>
Date: Tue, 9 Nov 2021 17:54:01 -0500
Subject: [PATCH] Use c++14 for TF 2.7

---
 source/CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index f05ab8df1..a3d62c09b 100644
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -46,7 +46,6 @@ endif(GIT_FOUND)
 
 # global defines
 list (APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/)
-set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-ignored-attributes")
 
 # model version
 file(READ ${PROJECT_SOURCE_DIR}/config/MODEL_VER MODEL_VERSION)
@@ -81,6 +80,12 @@ endif (USE_ROCM_TOOLKIT)
 # find tensorflow, I need tf abi info
 find_package(tensorflow REQUIRED)
 
+if (TENSORFLOW_VERSION GREATER_EQUAL 2.7)
+  set (CMAKE_CXX_STANDARD 14)
+else()
+  set (CMAKE_CXX_STANDARD 11)
+endif()
+
 # find threads
 find_package(Threads)
 
